### DISCUSSION TITLE
How do I Show Pathfinding?

### DISCUSSION THREAD

**Discussion by Yomatsusshimano** [Courseplay/Courseplay_FS22#1733] [Q&A]
Good evening, 

I have enabled debug mode in the vehicle settings, enabled all debug modes within the Shift+4 menu, and enabled development mode within the game.xml file. I cannot get the pathfinding to show up like [WMAvant](https://github.com/WMAvant) got it to in this screenshot.
![173202707-73ccec91-03c1-40f6-9fae-cde157b0dc6f](https://user-images.githubusercontent.com/5701795/176986600-131b04e4-b5dc-46c3-92e5-6e7536af343f.png)

What am I missing? I am trying to report an issue but I can't get the necessary information. I also am having trouble getting a debug print to populate. 

Courseplay 7.1.0.0

Thanks for any help!

**Comment by Tensuko** [comment]
You only have to turn on the debug for vehicle in vehicle setting, then run it and post logfile.

What is the Issue ?

**Reply by Yomatsusshimano** [reply]
Hmm, okay yeah I do have it turned on. I'm running into an issue with the Case Cotton harvester saying "Blocked by object" after it auto-unloads a cotton bale. It doesn't do it with every bale. I haven't identified the exact scenario that causes it yet.

**Reply by Tensuko** [reply]
And why do you think it got anything to do with Pathfinder?

Thats just the case when the Bale blocks the vehicle.
Screenshot when its blocked ?

**Reply by Tensuko** [reply]
Oh and your screenshot, I am pretty sure that JohnDeere is a mod!
Seems like converted one I guess with the old FS19 collision.
That needs to be removed. Ppl should stick to Giants Standart for FS22.

**Reply by Yomatsusshimano** [reply]
Here's a top down view of one of the stucks. I wasn't sure which F5 overlay to use. I just used the one that appears so show any obstacles. Next time it happens I can send a different more specific screenshot if need be. 
![Stuck](https://user-images.githubusercontent.com/5701795/176993443-ed4cb6f1-d37c-4d3a-9335-9ac48f36a989.png)

And yeah, I'm not quite sure what is being used in that first screenshot. I just happened to find it in the following issue thread when trying to look for answers. https://github.com/Courseplay/Courseplay_FS22/issues/1700

**Reply by Yomatsusshimano** [reply]
If it matters, I am logged in as admin on a hosted server.

**Reply by Yomatsusshimano** [reply]
This is what I have set for the debug settings. My debug print folder is however empty so I must be missing something.
![activated](https://user-images.githubusercontent.com/5701795/176993569-8f5f7a44-6097-45ad-a704-b3990ba96ed2.png)
![debug](https://user-images.githubusercontent.com/5701795/176993570-976d5f7b-f3cc-43e1-9879-14c96608052d.png)

**Reply by Yomatsusshimano** [reply]
Here is a different angle if that helps.
![fsScreen_2022_06_30_23_23_58](https://user-images.githubusercontent.com/5701795/176993689-3b8a0174-e416-4777-9411-db5f1663f054.png)

**Reply by Tensuko** [reply]
> My debug print folder is however empty so I must be missing something

That folder is just for us, the normal logfile should have all prints in it.
But as I guessed somehow the bale stops him, not sure why that is.

Will have to investigate that.

**Reply by Yomatsusshimano** [reply]
Yeah, I'm unsure if it does it with the round bales. I'll have to test that eventually.

**Reply by Yomatsusshimano** [reply]
After some testing, I believe it happens when a bale is unloaded while the harvester is on a decline.
